engine/class_modules/paladin/sc_paladin_protection.cpp

```diff
@@ -315,23 +308,21 @@ struct judgment_prot_t : public judgment_t
     cooldown -> duration *= 1.0 + p -> spec.protection_paladin -> effectN( 5 ).percent();
   }
 
+  private:
```

Comment:
That value should be set during construction (it also doesn't really need to be specified private)